### PR TITLE
Add BigQuery scopes to GCP [BA-5716]

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

**Comment by geoffjentry** [comment]
@mcovarr Yeah, I discussed with @ruchim - we're skipping that for now, but agreed that it'd be a nice idea in general (tho i'd say `workflow_options`, not config)

**Comment by cjllanwarne** [comment]
Did we ever document what scope you need to allow on a project for it to be used with Cromwell? If so, we presumably need to add these to it

**Comment by geoffjentry** [comment]
@cjllanwarne I don't understand. This is adding the required scope.

**Comment by ruchim** [comment]
Hello Jeff—would you mind testing if this would break Terra or Green team prod? Meaning, if the SAs don’t have BQ scope before being sent to Cromwell, and then Cromwell adds the BQ scope—does the task fail to run?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by dinvlad** [comment]
Btw, according to https://cloud.google.com/bigquery/docs/reference/rest/v2/tabledata/insertAll, it's not necessary to add `BIGQUERY_INSERTDATA` scope if you already have `BIGQUERY` scope.

File: supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/GenomicsFactory.scala [hunk 1]
```diff
@@ -154,6 +155,9 @@ case class GenomicsFactory(applicationName: String, authMode: GoogleAuthMode, en
             Oauth2Scopes.USERINFO_PROFILE,
             // Monitoring scope as POC
             GenomicsFactory.MonitoringWrite,
+            // Allow read/write with BigQuery
+            BigqueryScopes.BIGQUERY,
+            BigqueryScopes.BIGQUERY_INSERTDATA
```
Comment (dinvlad) [inline supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/GenomicsFactory.scala]: NB: BigqueryScopes.BIGQUERY_INSERTDATA is unnecessary here

**Comment by geoffjentry** [comment]
@mcovarr @ruchim I confirmed that this is fine. The underlying PAPI job will fail as expected (since it does not have rights to access BQ) but nothing blows up in a weird way. 

Steps:

1. Make a minimally permissioned SA and try running. This actually succeeds as the default compute service account has editor access. However, nothing blew up when setting the BQ scope. Sorta success.
2. Use that minimally permissioned SA as the `google_compute_service_account` - the PAPI job fails as expected, but nothing else untoward happened

**Pull request merged**